src/frontend/packages/store/src/entity-request-pipeline/pipline-http-client.service.ts

```diff
@@ -4,14 +4,12 @@ import { Store } from '@ngrx/store';
 import { Observable } from 'rxjs';
 import { filter, first, map, mergeMap } from 'rxjs/operators';
 
-import { environment } from '../../../core/src/environments/environment';
 import { InternalAppState } from '../app-state';
 import { StratosCatalogEndpointEntity } from '../entity-catalog/entity-catalog-entity/entity-catalog-entity';
 import { IStratosEndpointDefinition } from '../entity-catalog/entity-catalog.types';
+import { cfAPIVersion, proxyAPIVersion } from '../jetstream';
 import { connectedEndpointsOfTypesSelector, endpointOfTypeSelector } from '../selectors/endpoint.selectors';
 
-const { proxyAPIVersion, cfAPIVersion } = environment;
```

Comment:
To remove duplication both of these properties should be removed from `environment`